addons/mail/static/src/owl/components/composer/composer.js

```diff
@@ -303,14 +309,19 @@ class Composer extends Component {
      * @param {string} ev.detail.unicode
      */
     _onEmojiSelection(ev) {
-        this._textInputRef.comp.insertTextContent(ev.detail.unicode);
+        this._saveComposerContent();
+        this.storeDispatch(
+            'insertEmoji',
```

Comment:
suggestion: rename action to `insertEmojiInComposerTextInput`
also would prefer `composerLocalId` before emoji unicode: this is an action on composer, and emoji unicode is some data

Reply:
Done